### PR TITLE
Add AMP Settings label for gallery block toggle settings

### PR DESCRIPTION
**Pull request by thelovekesh**
## Summary

Add a label for gallery block AMP carousel and lightbox toggle settings.

image

## Checklist

- [x] My code is tested and passes existing [tests](https://github.com/ampproject/amp-wp/wiki/Engineering-Guidelines#tests).
- [x] My code follows the [Engineering Guidelines](https://github.com/ampproject/amp-wp/wiki/Engineering-Guidelines) (updates are often made to the guidelines, check it out periodically).

### CODE REVIEW

File: assets/src/block-editor/helpers/index.js [hunk 1]
```diff
@@ -344,8 +344,13 @@ ImageBlockLayoutAttributes.propTypes = {
  */
 const GalleryBlockLayoutAttributes = (props) => (
 	<InspectorAdvancedControls>
-		<AmpLightboxToggle {...props} />
-		<AmpCarouselToggle {...props} />
+		<div>
+			<p className="components-base-control__label">
```
Comment (westonruter) [inline assets/src/block-editor/helpers/index.js]: I guess there's no component specifically for this?

Comment (thelovekesh) [reply assets/src/block-editor/helpers/index.js]: Unable to find such on https://wordpress.github.io/gutenberg/.

Comment (thelovekesh) [reply assets/src/block-editor/helpers/index.js]: Tried `<Placeholder />` but it creates a border with a box and doesn't fit. See working at https://wordpress.github.io/gutenberg/?path=/story/components-placeholder--default

Comment (westonruter) [reply assets/src/block-editor/helpers/index.js]: What about `BaseControl.VisualLabel`? 

https://github.com/WordPress/gutenberg/blob/64a873a80b001a1c2c892c0c4a56010a94cc4c03/packages/components/src/base-control/index.tsx#L101-L135
https://wordpress.github.io/gutenberg/?path=/story/components-basecontrol--with-visual-label

Comment (westonruter) [reply assets/src/block-editor/helpers/index.js]: Maybe that has to be inside of a `BaseControl`?

Comment (thelovekesh) [reply assets/src/block-editor/helpers/index.js]: Yes, it works. Thanks 🙌🏼 

image

**Review by westonruter — APPROVED** [review]

**Comment by pavanpatil1** [comment]
QA passed ✅ 

Cross-checked the issue, Now the label is properly visible for the AMP settings for the gallery block in advance settings option. 

image

**Pull request merged**